src/scripts/calculate-winner.js

```diff
@@ -0,0 +1,20 @@
+export const winLines = [
+    [0, 1, 2],
+    [3, 4, 5],
+    [6, 7, 8],
+    [0, 3, 6],
+    [1, 4, 7],
+    [2, 5, 8],
+    [0, 4, 8],
+    [2, 4, 6],
+];
+
+export function calculateWinner(squares) {
+    for (let i = 0; i < winLines.length; i++) {
+        const [a, b, c] = winLines[i];
+        if (squares[a] && squares[a] === squares[b] && squares[a] === squares[c]) {
+            return squares[a];
+        }
+    }
+    return squares.indexOf(null) < 0 ? 'tie' : null;
```

Comment:
in this case it's better to do the check not in this method, because `tie` is not a winner

Reply:
```suggestion
const {
            history,
            stepNumber,
            xIsNext,
            isGameOver
        } = this.state;
        const currentSquares = history[stepNumber].squares;
        const winner = calculateWinner(currentSquares);
        const status = winner ? `Winner is ${winner}`
            : ((history.length - 1 === stepNumber && isGameOver) ? 'Draw' : `Next move: ${xIsNext ? GAME_SYMBOLS.X : GAME_SYMBOLS.O}`);
```
This is code from render() method of Game component. Fixed to it, now `calculate-winner` method returns only winner or null.